acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/publickeys/InterlockKeysAcceptanceTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.tests.publickeys;
+
+import static org.hamcrest.Matchers.containsInAnyOrder;
+
+import tech.pegasys.teku.bls.BLSSecretKey;
+import tech.pegasys.web3signer.core.signing.KeyType;
+import tech.pegasys.web3signer.dsl.utils.MetadataFileHelpers;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import io.restassured.response.Response;
+import org.apache.tuweni.bytes.Bytes32;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+@Disabled("Requires physical access to Interlock on USB Armory II")
+public class InterlockKeysAcceptanceTest extends KeyIdentifiersAcceptanceTestBase {
+  private static final MetadataFileHelpers metadataFileHelpers = new MetadataFileHelpers();
+
+  // following keys are expected to be pre-loaded in USB Armory/Interlock under path
+  // /bls/key<1..n>.txt
+  private static final List<String> PRE_LOADED_KEYS =
```

Comment:
assuming these are the private keys?

Reply:
yes they are, will update variable name to represent such.